stages/org.osbuild.sshd.config

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+"""
+Configure the SSH daemon.
+"""
+
+import fileinput
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["config"],
+"properties": {
+  "config": {
+    "additionalProperties": false,
+    "description": "SSHD config options",
+    "type": "object",
+    "properties": {
+      "password_authentication": {
```

Comment:
I wonder about the config names, since in the ssh config they are all CamelCase. In the `oci-archive` stage I have went the rational that these keys kinda belong to the container spec and just used them verbatim, i.e. via CamelCase. But I am unsure.

Reply:
Changed.